README.md

```diff
@@ -90,6 +90,8 @@ Then build the code using `go build` which will produce the built binary in a fi
 
 `--spot-node-label` (default: `node-role.kubernetes.io/spot-worker`) Name of label on nodes to be considered as targets for pods.
 
+`--delete-non-replicated-pods` (default: `false`) Delete non replicated pods in on-demand instance for rescheduling.
```

Comment:
Can we change this message to `Delete non-replicated pods running on on-demand instances`

Technically some non-replicated pods won't actually be rescheduled so I'd prefer we avoid suggesting they will.

Reply:
Changed message and added note that some no-replicated pods won't be rescheduled